jsHelper/spicetifyWrapper.js

```diff
@@ -574,8 +593,13 @@ window.Spicetify = {
 	}
 
 	const dropdownChunk = Object.entries(require.m).find(([, value]) => value.toString().includes("dropDown") && value.toString().includes("isSafari"));
-	if (dropdownChunk) {
-		Spicetify.ReactComponent.Dropdown = Object.values(require(dropdownChunk[0])).find(m => typeof m === "function");
+	if (dropdownChunk) Spicetify.ReactComponent.Dropdown = Object.values(require(dropdownChunk[0])).find(m => typeof m === "function");
```

Comment:
there is no reason to make this one line
you just make readability worse

Reply:
you literally said if it's in one line then it's fine, now you change your mind

Reply:
because this case is special https://github.com/spicetify/spicetify-cli/pull/2792#issuecomment-1920174012